gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -347,7 +348,7 @@ def preprocess_kerchunk(cls, refs: dict) -> dict:
                 ref_names.add(re.match(file_match_pattern, ref).group(1))
         for ref in ref_names:
             fill_value_fix = json.loads(refs[f"{ref}/.zarray"])
-            fill_value_fix["fill_value"] = str(cls.missing_value)
+            fill_value_fix["fill_value"] = str(cls.missing_value)  # TODO: Do we need `str`? Gets dumped to json below
```

Comment:
Again, I agree this `str` is superfluous

Reply:
4b82f85